com.google.Earth.json

```diff
@@ -0,0 +1,94 @@
+{
+  "app-id": "com.google.Earth",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "18.08",
+  "sdk": "org.freedesktop.Sdk",
+  "command": "google-earth",
+  "separate-locales": false,
+  "finish-args": [
+    "--share=ipc",
+    "--socket=x11",
+    "--share=network",
+    "--device=dri",
+    "--filesystem=home",
+    "--talk-name=org.freedesktop.secrets",
+    "--env=BROWSER=xdg-open"
+  ],
+  "modules": [
+    {
+      "name": "glu",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://mesa.freedesktop.org/archive/glu/glu-9.0.0.tar.bz2",
+          "sha256": "1f7ad0d379a722fcbd303aa5650c6d7d5544fde83196b42a73d1193568a4df12"
+        }
+      ],
+      "cleanup": [ "/include", "/lib/*.a", "/lib/*.la", "/lib/pkgconfig" ]
+    },
+    {
+      "name": "google-earth",
+      "buildsystem": "simple",
+      "build-commands": [
+        "install -Dm755 apply_extra /app/bin/apply_extra",
+        "install -Dm755 google-earth /app/bin/google-earth",
+        "install -Dm644 com.google.Earth.appdata.xml /app/share/appdata/com.google.Earth.appdata.xml",
+        "install -Dm644 product_logo_64.png /app/share/icons/hicolor/64x64/com.google.Earth.png",
+        "install -Dm644 product_logo_128.png /app/share/icons/hicolor/128x128/com.google.Earth.png",
+        "install -Dm644 product_logo_256.png /app/share/icons/hicolor/256x256/com.google.Earth.png",
+        "cp /usr/bin/ar /app/bin",
+        "ARCH_TRIPLE=$(gcc --print-multiarch) && cp /usr/lib/${ARCH_TRIPLE}/libbfd-*.so /app/lib"
+      ],
+      "sources": [
+        {
+          "type": "extra-data",
+          "filename": "google-earth.deb",
+          "only-arches": ["x86_64"],
+          "url": "https://dl.google.com/dl/earth/client/current/google-earth-pro-stable_current_amd64.deb",
+          "sha256": "f726869215993755b7ce8ea34fc5c298346ab0b830586cc9213c08b9f621b3ea",
+          "size": 56514550
+        },
+        {
+          "type": "script",
+          "dest-filename": "apply_extra",
+          "commands": [
+              "ar x google-earth.deb",
+              "rm -f google-earth.deb",
+              "tar xf data.tar.xz",
+              "rm -f control.tar.gz data.tar.xz debian-binary",
+              "mv opt/* .",
+              "mv usr/* .",
+              "chmod -R a-s,go+rX,go-w .",
+              "rmdir usr",
+              "mkdir -p export/share/applications",
+              "sed 's|Icon=google-earth-pro|Icon=com.google.Earth|; s|Exec=/opt/google/earth/pro/google-earth-pro|Exec=google-earth|' /app/extra/google/earth/pro/google-earth-pro.desktop > export/share/applications/com.google.Earth.desktop",
```

Comment:
I think it's better to add the .desktop file to the repo (for appdata categories).

Reply:
How do you mean? This is the upstream desktop file. I should probably be using desktop-file-edit though.

Reply:
I mean, I would add the .desktop to this flathub repo, because appstream-compose uses it to add categories to the appdata file.